plugins/woocommerce-blocks/assets/js/blocks/product-gallery/edit.tsx

```diff
@@ -88,49 +85,13 @@ const TEMPLATE: InnerBlockTemplate[] = [
 	],
 ];
 
-const getMode = ( currentTemplateId: string, templateType: string ) => {
-	if (
-		templateType === 'wp_template_part' &&
-		currentTemplateId.includes( 'product-gallery' )
-	) {
-		return 'full';
-	}
-	return 'standard';
-};
-
 export const Edit = ( {
 	clientId,
 	attributes,
 	setAttributes,
 }: BlockEditProps< ProductGalleryAttributes > ) => {
 	const blockProps = useBlockProps();
 
-	const { currentTemplateId, templateType } = useSelect( ( select ) => {
-		const store = select( 'core/edit-site' );
-		return {
-			currentTemplateId: store ? store.getEditedPostId() : '',
-			templateType: store ? store.getEditedPostType() : '',
-		};
-	}, [] );
-
-	useEffect( () => {
-		const mode = getMode( currentTemplateId, templateType );
-
-		setAttributes( {
-			...attributes,
-			mode,
-			productGalleryClientId: clientId,
-		} );
-		// Move the Thumbnails block to the correct above or below the Large Image based on the thumbnailsPosition attribute.
-		moveInnerBlocksToPosition( attributes, clientId );
```

Comment:
Removing this breaks Thumbnails positioning. You should keep it for and I'll get rid of it in a Thumbnails cleanup